### ISSUE TITLE
Renaming a folder keeps the older name as a suffix

### ISSUE THREAD

**Issue by CosminGramada** [riok/Kreya#147]
**Describe the bug**
When a folder is renamed, the previous name remains as a suffix on the disk and tab name.

**To Reproduce**
1. Open Kreya app
2. Select a folder, right click and select New Folder
3. Give the new folder a name
4. Rename the folder
5. Check the tab name and folder name on the disk

**Expected behavior**
Only the new name is displayed both as the tab name and on the disk.

**Environment (if possible, copy the information from the error dialog or the About menu):**
Apple M1 Pro
OS: MacOS 13.2.1
Kreya Version 1.10.1

![gif Mar-25-2023 12-03-01](https://user-images.githubusercontent.com/15174660/227710679-d6a5cbc4-518d-4ab7-8c62-807334ac0ee5.gif)

LE:
Ran into an app error having the root cause the issue above:

```
{
  "kreyaVersion": "1.10.1",
  "releaseChannel": "stable",
  "osDescription": "Darwin 22.3.0 Darwin Kernel Version 22.3.0: Mon Jan 30 20:38:37 PST 2023; root:xnu-8792.81.3~2/RELEASE_ARM64_T6000",
  "osVersion": "Unix 13.2.1",
  "osArch": "Arm64",
  "processArch": "Arm64",
  "runtimeIdentifier": "osx.13-arm64",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}

System.IO.DirectoryNotFoundException: Could not find a part of the path '/Users/cosmingramada/Desktop/Edus/Tests/EdusTests/2. Organizations/3. Update/2. Negative/2. Update organization name using existing value. Update organization name using existing value/1. Create new organization.js'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Func`4 createOpenException)
   at System.IO.File.WriteToFileAsync(String path, FileMode mode, String contents, Encoding encoding, CancellationToken cancellationToken)
   at Kreya.Data.FileSystem.FileService.WriteText(String path, String text) in /builds/riokch/kreya/kreya/src/Kreya.Data/FileSystem/FileService.cs:line 100
   at Kreya.Data.OperationScriptStorage.StoreScript(IOperationWithDetail operation) in /builds/riokch/kreya/kreya/src/Kreya.Data/OperationScriptStorage.cs:line 42
   at Kreya.Data.OperationStorage.StoreOperationDetail(Project project, IOperationWithDetail operation) in /builds/riokch/kreya/kreya/src/Kreya.Data/OperationStorage.cs:line 124
   at Kreya.Core.Operations.OperationService.StoreOperation(IOperationWithDetail operation) in /builds/riokch/kreya/kreya/src/Kreya.Core/Operations/OperationService.cs:line 137
   at SpiderEye.Bridge.ApiMethod.InvokeAsync(Object instance, Object parameter)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.InvokeWithDependencyInjection(IDependencyInjectionApiMethod apiMethod, Object parameters)
   at SpiderEye.Bridge.WebviewBridge.ResolveCall(String id, String parameters)
```

**Comment by CommonGuy** [comment]
Thank you for reporting this bug and the excellt GIF showing the problem. We will look into this

**Comment by CommonGuy** [comment]
This has been fixed with Kreya 1.11, which we just released
